lib/simulator-xcode-6.js

```diff
@@ -284,43 +284,43 @@ class SimulatorXcode6 extends EventEmitter {
     } catch (ign) {
       // ignore error
     }
-
-    let simulatorApp = path.resolve(await getXcodePath(), 'Applications', this.simulatorApp);
-    let args = ['-Fn', simulatorApp, '--args', '-CurrentDeviceUDID', this.udid];
     opts = Object.assign({
       scaleFactor: null,
       connectHardwareKeyboard: false,
       allowTouchEnroll: false,
       startupTimeout: this.startupTimeout,
     }, opts);
-
-    if (opts.scaleFactor) {
-      const supportedScales = ['1.0', '0.75', '0.5', '0.33', '0.25'];
-      if (supportedScales.indexOf(opts.scaleFactor) < 0) {
-        log.errorAndThrow(`Only "${supportedScales}" values are supported as scale factors. "${opts.scaleFactor}" is passed instead.`);
-      }
-      const stat = await this.stat();
-      const formattedDeviceName = stat.name.replace(/\s+/g, '-');
-      const argumentName = `-SimulatorWindowLastScale-com.apple.CoreSimulator.SimDeviceType.${formattedDeviceName}`;
-      args.push(argumentName, opts.scaleFactor);
-    }
-
-    if (!opts.connectHardwareKeyboard) {
-      args.push('-ConnectHardwareKeyboard', '0');
-    }
-
+    const startTime = process.hrtime();
     // Set the 'Touch ID Enroll' key bindings before the Simulator starts
     if (opts.allowTouchEnroll) {
       await setTouchEnrollKey();
     }
+    if (await this.isUIClientRunning()) {
+      await simctl.bootDevice(this.udid);
```

Comment:
This is ok for other simulators as well, since this.shutdown() kills the UI client as well prior  toXcode9, which means this condition branch is never going to be executed there.